ssl/ssl_transfer_asn1.cc

```diff
@@ -930,6 +930,10 @@ static const unsigned kSSLConfigOcspStaplingEnabledTag =
 static const unsigned kSSLConfigJdk11WorkaroundTag =
     CBS_ASN1_CONSTRUCTED | CBS_ASN1_CONTEXT_SPECIFIC | 1;
 
+static const unsigned kSSLConfigConfMaxVersionUseDefault = 2;
```

Comment:
Default tagging is `EXPLICIT` per https://github.com/aws/aws-lc/blob/main/ssl/tls_transfer.asn#L1. Hence this needs `CBS_ASN1_CONSTRUCTED` and `CBS_ASN1_CONTEXT_SPECIFIC`.